src/OneExplorer/ConfigObject.ts

```diff
@@ -140,6 +148,10 @@ export class ConfigObj {
     this.uri = uri;
     this.rawObj = rawObj;
     this.configType = CfgType.one;
+    const ext = path.extname(uri.fsPath);
+    if (ext === EdgeTpuConfigSetting.ext) {
+      this.configType = CfgType.edgeTpu;
+    }
```

Comment:
```suggestion
    const ext = path.extname(uri.fsPath);
    switch(ext) {
      case EdgeTpuConfigSetting.ext:
        this.configType = CfgType.edgeTpu;
        break;
      case OneConfigSetting.ext:
      default:
        this.configType = CfgType.one;
    }
```

I meant, like this..?

Reply:
Thank you. I understand it. I will apply this :-D